backend/src/services/event_service.py

```
@@ -1413,6 +1439,37 @@ def update(
+                if new_attendance is None:
+                    updates["attendance"] = "planned"
+                    logger.debug(f"Reverting attendance to 'planned' (moved from forces_skip status '{old_status}')")
+        elif new_attendance and new_attendance != "skipped":
+            # Attendance is changing without a status change - check current status
+            effective_status = new_status or old_status
+            if self._status_forces_skip(effective_status, team_id):
+                raise ValidationError(
+                    f"Cannot change attendance away from 'skipped' while status "
+                    f"'{effective_status}' forces skip",
+                    field="attendance"
+                )
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)